src/index.js

```diff
@@ -1,13 +1,74 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
+class TodoData {
+  constructor() {
+    if (window.localStorage.getItem('tasksData') !== undefined) {
+      this.tasksData = JSON.parse(window.localStorage.getItem('tasksData'));
+    } else {
+      this.tasksData = [];
+    }
   }
```

Comment:
- [ ] Dear @EnixWord, I clone your project and install all the packages. but I could not add a task which means I could not test the functionality for `editing` and `deleting` tasks. Please make sure your app is capable of storing tasks. 
- [ ] Please notice, All changes to the To-Do List should be saved in local storage.
#### make sure the following ar working, please.
> A function for adding a new task
> A function for editing task descriptions
> A function for deleting a task

https://user-images.githubusercontent.com/69077061/221200572-590a578e-f774-4e5b-87ee-29addcc46347.mp4

- [ ] Also make sure that the `completed` is set to `false` by default and the property `index` is set to the value of the new array length.